src/device-registry/routes/api-v1.js

```diff
@@ -2747,6 +2747,59 @@ router.get(
   siteController.findNearestSite
 );
 
+router.post(
+  "/sites/approximate",
+  oneOf([
+    [
+      body("latitude")
+        .exists()
+        .withMessage("the latitude should be provided")
+        .bail()
+        .matches(constants.LATITUDE_REGEX, "i")
+        .withMessage("the latitude provided is not valid")
+        .bail()
+        .custom((value) => {
+          let dp = decimalPlaces(value);
+          if (dp < 5) {
+            return Promise.reject(
+              "the latitude must have 5 or more characters"
+            );
+          }
+          return Promise.resolve("latitude validation test has passed");
+        })
+        .bail()
+        .customSanitizer((value) => {
+          return numeral(value).format("0.00000000000000");
```

Comment:
Sanitising to 14 decimal places in the GPS coordinates input validation for the new approximation endpoint. "The more the decimal places, the better"